torch/ao/quantization/fx/convert.py

```diff
@@ -124,7 +124,7 @@ def _replace_observer_with_quantize_dequantize_node(
             graph.erase_node(node)
         return
 
-    # otherwise, we can convert the observer module call to quantize/dequantize node
+    # otherwise, we can convert the activation_post_process module call to quantize/dequantize node
```

Comment:
`observer` made sense, not sure why `activation_post_process` is in the comment?  That seems to be just a variable name, not a concept known throughout the AO codebase

Reply:
we use `activation_post_process` in the code base to represent observer or fake quant module, I think we should probably just rename this to observer_or_fake_quant at some point..